Assets/MixedRealityToolkit/Definitions/Utilities/ArticulatedHandPose.cs

```diff
@@ -181,6 +177,10 @@ public enum GestureId
             /// Victory sign
             /// </summary>
             Victory,
+            /// <summary>
+            /// Relaxed hand pose, grab point does not move
```

Comment:
nit: since we're here, should we start labeling each value instead of auto-increment i.e
Victor = x,
OpenSteadyGrabPoint = y 

and make this trend for all other enums going forward?

Reply:
I can take a separate pass for existing enums, but yeah, this would be good to do in the future